Build error with stat_tool and sequence Analysis

@guedon 

src/cpp/stat_funs3.cpp:3888:19: error: no member named 'select_step' in 'stat_tool::Vectors'
src/cpp/stat_funs3.cpp:3901:19: error: no member named 'select_step' in 'sequence_analysis::Sequences'
src/cpp/stat_funs3.cpp:3917:29: error: no member named 'select_step' in 'sequence_analysis::MarkovianSequences'
...

--- a/aml/src/wrapper/SConscript
+++ b/aml/src/wrapper/SConscript
@@ -1,7 +1,7 @@
 # -*-python-*-
 
 Import( "env" )
-import os 
+import os
 pj = os.path.join
 
 sources = env.ALEAGlob('*.cpp')
@@ -30,7 +30,7 @@ pglsg""")
 # MTG module
 LIBS += ["vpmtg"]
 # TreeMatching module
-LIBS += ["vptree_matching"]
+#LIBS += ["vptree_matching"]
 # # MSVOXEL module
 # LIBS += ["vpoctree"]
 # #  FLUX module

